### DISCUSSION TITLE
Cant seem to use cast on my terminal.

### DISCUSSION THREAD

**Discussion by ikennaumeh** [Cyfrin/foundry-full-course-cu#644] [Q&A] [answered]
So after i round the foundryup in my terminal, i get anvil and forge, and i'm able to check their versions too, but when i run
```
cast --version
```
I get :
```
bash: /home/ikennaumeh/.foundry/bin/cast: cannot execute binary file: Exec format error
```

**Comment by theirrationalone** [comment, marked as answer]
go here 👎

```terminal
\\wsl.localhost\Ubuntu\home\theirrationalone\.foundry\bin
```
and delete all and then try `foundryup` again.

**Reply by ikennaumeh** [reply]
how do i get there please

**Reply by theirrationalone** [reply]
if you're on windows...

This Pc -> on right side menu below there a new linux instance icon has appeared -> click on it -> click on Ubuntu folder -> click on home folder -> click on your username folder -> click on .foundry folder -> click on bin -> delete everything except foundryup

![linux pathdf](https://github.com/Cyfrin/foundry-full-course-f23/assets/96755510/5a849c01-93a8-4777-bd9e-beb737d81913)

**Reply by ikennaumeh** [reply]
Thanks alot. Fixed now.
